.env.example

```diff
@@ -7,20 +7,41 @@
 
 # Environment. Valid options are development, stage or production
 WP_ENV=development
-#WP_CLI_HOME=local.url
-MYSQLI_DEFAULT_SOCKET=false
+#WP_CLI_HOME=local.site # optional
 
-# API Database
+### API Database
+#DB_HOST=localhost # optional
+#DB_PREFIX=wp_ # optional
 DB_NAME=local
 DB_USER=root
 DB_PASSWORD=root
-WP_HOME=http://site.local
+MYSQLI_DEFAULT_SOCKET=false # optional
+
+### Home & URL
+WP_HOME=https://local.site
 WP_SITEURL=${WP_HOME}/wp
-# Optional variables
-#DB_HOST=localhost
-#DB_PREFIX=wp_
 
-# Generate your keys here: https://roots.io/salts.html
+### Network Setup
+#WP_ALLOW_MULTISITE=true # optional
+#IS_MULTISITE=1 # optional: 0 = single site, 1 = multisite
+#SUBDOMAIN_INSTALL=false # optional: set up multisite with sub directory or sub domain, only used if IS_MULTISITE=1
```

Comment:
@Sisanu Change comment to : # optional: set up multisite with sub directory or sub domain, only used if WP_ALLOW_MULTISITE=true